src/library-authoring/data/apiHooks.ts

```diff
@@ -135,7 +137,6 @@ export const useCreateLibraryBlock = () => {
   return useMutation({
     mutationFn: createLibraryBlock,
     onSettled: (_data, _error, variables) => {
-      queryClient.invalidateQueries({ queryKey: libraryAuthoringQueryKeys.contentLibrary(variables.libraryId) });
```

Comment:
@rpenido One question here, Is it not necessary to invalidate the library query when creating a block?

Reply:
You are right @ChrisChV!
Reverted here: c9b0c9eb530bd0a9de86625412ed666044b5ccd4